cli/flox/src/commands/activate.rs

```diff
@@ -341,11 +351,13 @@ impl Activate {
         cmd.arg(log_path);
         cmd.env("_FLOX_WATCHDOG_LOG_LEVEL", "debug"); // always write to log file
```

Comment:
nit: could be configurable, i.e. overridable by parent env, otherwise why do we default to `debug`?

Reply:
@zmitchell can you describe what you were thinking here?

Reply:
Because otherwise you get an empty log file